tests/laser/state/mstack_test.py

```diff
@@ -0,0 +1,68 @@
+from mythril.laser.ethereum.state import MachineStack
+from mythril.laser.ethereum.evm_exceptions import *
+from tests import BaseTestCase
+
+
+class MachineStackTest(BaseTestCase):
+
+    @staticmethod
+    def test_mstack_constructor():
+        mstack = MachineStack([1, 2])
+        assert(mstack == [1, 2])
+
+        mstack = MachineStack([])
+        assert(mstack == [])
+
+    @staticmethod
+    def test_mstack_append():
+        mstack = MachineStack([])
+        mstack.append(0)
+        assert(mstack == [0])
+
+        for i in range(mstack.STACK_LIMIT-1):
+            mstack.append(1)
+
+        try:
+            mstack.append(1000)
+        except StackOverflowException:
```

Comment:
You can use pytest.raises here, like in mstate_tests

Reply:
Oh, these tests aren't written using pytest. I think pytests provides some really nice features, like the one mentioned here.
Let me know if you disagree, but lets write all the (new) tests using pytest (because uniformity + functionality)

Reply:
looks like pytest.raises works without any pytest stuff. I think using either of one is fine(both have their own list of good and bad), so let's stick to a convention and start using that from now on (pytest)